src/main/java/study/step4/domain/Cars.java

```diff
@@ -0,0 +1,39 @@
+package study.step4.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    public Cars winners() {
+        List<Car> winners = new ArrayList<>();
+        for (Car car: cars) {
+            addWinner(winners, car);
+        }
+        return new Cars(winners);
+    }
+
+    private void addWinner(List<Car> winners, Car car) {
+        if (car.position().getPosition() == maxPosition()) {
+            winners.add(car);
+        }
+    }
```

Comment:
값을 꺼내서 사용하지 않고 메세지를 보내어 객체지향으로 구현해보시는건 어떠신가용?

```suggestion
    private void addWinner(List<Car> winners, Car car) {
        if (car.isWinner(maxPosition())) {
            winners.add(car);
        }
    }
```